packages/test-commons/src/utils.ts

```diff
@@ -34,16 +34,14 @@ export function dispatchTestKeyboardEvent(target: EventTarget, eventType: string
     target.dispatchEvent(event);
 }
 
-/**
- * Enum of possible browsers
- */
+/** Enum of known browsers */
 enum Browser {
-    CHROME = "Chrome",
-    EDGE = "Edge",
-    FIREFOX = "Firefox",
-    IE = "IE",
-    UNKNOWN = "unknown",
-    WEBKIT = "WebKit",
+    CHROME,
+    EDGE,
+    FIREFOX,
+    IE,
+    UNKNOWN,
+    WEBKIT,
```

Comment:
why an integer enum?

Reply:
it's entirely internal to this file and i declared in the description that actual `enum` types should prefer numbers unless they have a good reason for strings.